src/features/flows/nRF9151/SimCard.tsx

```diff
@@ -13,40 +13,51 @@ import { Next } from '../../../common/Next';
-                    to use within the SIMs lifetime. In addition, you will get a
-                    two-month trial access to Onomondos real-time network
-                    insight tool.
-                </li>
-                <br />
-                <li>
-                    <Link
-                        label="SIM card from Wireless Logic"
-                        href="https://www.wirelesslogic.com/simclaim/nsctrial/"
-                        color="tw-text-primary"
-                    />{' '}
-                    with 5 MB free data for 6 months.
-                    <br />
-                    If you register the SIM card within the trial period, you
-                    will get additional 45 MB to use within 5 years.
-                </li>
-            </ol>
-            <br />
-            <p>
-                Select the SIM card that is supported in your area (check the
-                links), plug it in, and it will work out of the box.
+        <Main.Content
+            heading="Plug in SIM card"
+            subHeading="There are two pre-activated SIM cards included that work out-of-the-box*"
+        >
+            <div>
+                <b>Onomondo</b>
+                <p>
+                    10 MB free data. If you register the SIM card, you will get
+                    additional 40 MB to use within the SIMs lifetime. In
+                    addition, you will get a two-month trial access to Onomondos
+                    real-time network insight tool.
+                </p>
+                <Link
+                    label="Register ownership"
+                    href="https://onomondo.com/go/nordic-dev-kit/#form"
+                    color="tw-text-primary"
+                />{' '}
+                |{' '}
+                <Link
+                    label="Check coverage"
+                    href="https://onomondo.com/go/nordic-dev-kit/#network"
+                    color="tw-text-primary"
+                />
+            </div>
+            <div className="tw-pt-8">
+                <b>Wireless Logic</b>
+                <p>
+                    5 MB free data for 6 months. If you register the SIM card
+                    within the trial period, you will get additional 45 MB to
+                    use within 5 years.
+                </p>
+                <Link
+                    label="Register ownership"
+                    href="https://www.wirelesslogic.com/simclaim/nsctrial/"
+                    color="tw-text-primary"
+                />{' '}
+                |{' '}
+                <Link
+                    label="Check coverage"
+                    href="https://www.wirelesslogic.com/simclaim/nsctrial/#COVERAGE"
+                    color="tw-text-primary"
+                />
+            </div>
+            <p className="tw-pt-8 tw-text-xs">
+                *As long as the SIM card has coverage in your area
             </p>
```

Comment:
Remove.

Reply:
Still needs removal. and integration with the header. This is a condition for proper usage of the card out of the box, so let's not hide it in some small text at the bottom as if this was a marketing asset.